skbio/alignment/_alignment.py

```diff
@@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
                                           "or fn is provided.")
 
         if ids is not None:
-            fn = lambda _: ids
```

Comment:
I did not know that was possible. Will update once I get travis back into a working state.